docker/web/Dockerfile

```diff
@@ -17,12 +17,12 @@
 # WARNING: Be careful about publishing images generated like this publicly
 # because your .git folder is exposed in the build step. We are not sure if
 # this is a security risk: stackoverflow.com/questions/56278325
-FROM maven:3.5.4 as build
+FROM maven:3-openjdk-11 as build
 COPY $PWD /cbioportal
 WORKDIR /cbioportal
 RUN mvn -DskipTests clean install
 
-FROM shipilev/openjdk-shenandoah:8
+FROM shipilev/openjdk-shenandoah:11
```

Comment:
looks good! Note to self: Java 12 has shenandoah by default https://openjdk.java.net/projects/jdk/12/

Reply:
> 
> 
> Thanks a lot for your contribution @nr23730 ! This looks great! We were just discussing upgrading to java 11. Curious if you were looking to upgrade to java 11 for a specific reason?
> 
> For us we noticed that container support is much better in later versions of java. Specifically we've been using the shenandoah GC for a while because we found it to perform better in our kubernetes setup. I saw that is included by default from java 12, currently we are using the backported images as i see you noticed in your PR. Another good reason for upgrading is that I noticed VS code will soon need java 11 to support latest java debugging features.
> 
> Travis is still running on java 8 (`.travis.yml`), but perhaps that's a good thing since it will take a while to upgrade our internal production servers to 8 and we prolly want to continue testing java 8 until those servers switched over to 11.
> 
> I did notice the localdb e2e test is failing so we'll have to investigate whether this is related to the 11 upgrade or not. It uses the docker image included in this repo, so it might be

I think there is a good reason for moving to Java 11 instead of a successor as Java 11 is a LTS (Long Term Support) Version (see: https://www.oracle.com/java/technologies/java-se-support-roadmap.html). The most recent version 14 was released in March and its support will end in September with the release of Java 15. 
For that reason, Java 11 is the way more commonly used version and also the default version of all recent Linux versions.

However, the step from Java 8 to Java 11 is huge as several libraries like JavaFX, Annotation or XML Bind were removed from the JDK itself. So running the App with Java 14 to get the enhanced garbage collection *should* be possible without any issues. This might be ensured using a CI test. ;)